pkg/cmd/mfa-recovery.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright © 2023 Doppler <support@doppler.com>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"github.com/DopplerHQ/cli/pkg/configuration"
+	"github.com/DopplerHQ/cli/pkg/http"
+	"github.com/DopplerHQ/cli/pkg/utils"
+	"github.com/spf13/cobra"
+)
+
+var mfaRecoveryCmd = &cobra.Command{
+	Use:   "mfa-recovery",
```

Comment:
Should we make the command `doppler mfa recovery` for more symmetry with the other options? (And in case we want to do other things with mfa in the future, e.g. allow the CLI to itself be an mfa option).

Reply:
You could also potentially nest it under `doppler login` (i.e. `doppler login mfa-recovery`).

Reply:
I like both of these options! I'll go with `doppler mfa recovery` as I like the idea of having more MFA commands in the future